features/admin-update-remove-publisher.feature

```diff
@@ -0,0 +1,22 @@
+@admin-update-remove-publisher @issue-381
+Feature: To remove publisher option in the admin
+  As a curator,
+  I do not want to see publisher option
+
+  Background:
+    Given Gigadb web site is loaded with production-like data
+
+
+  @ok @issue-381
+  Scenario: Guest user cannot visit admin dataset update page
+    Given I am not logged in to Gigadb web site
+    When I go to "/adminDataset/update/id/8"
+    Then I should see "Login"
+
+  @ok @issue-381
+  Scenario: Admin user can visit admin dataset update page and cannot see publisher option
+    Given I sign in as an admin
+    When I am on "/adminDataset/update/id/8"
+    And I should see "Update Dataset 100006"
+    And I should see a button input "Save"
+    Then I should not see "Publisher"
```

Comment:
I think it would be better if `admin-update-remove-publisher.feature` was incorporated into `dataset-admin.feature` since its tests are related to the dataset admin page. In fact, `Scenario: form loading with all necessary fields` will fail due to your work for this pull request because of this line :
```
And I should see a form element labelled "Publisher"
```

Maybe it is better to update `Scenario: form loading with all necessary fields` because you have removed the `Publisher` option.

Reply:
`admin-update-remove-publisher.feature` has been incorporated into `dataset-admin.feature` and `I should not see "Publisher"` scenario has been added into it.